src/pages/GridDefault/GridDefaultItem.tsx

```diff
@@ -0,0 +1,35 @@
+import './GridDefaultItem.scss';
+
+interface GridDefaultItemProps {
+  img: string;
+  title: string;
+}
+
+export const GridDefaultItem = (props: GridDefaultItemProps) => {
+  const { img, title } = props;
+  return (
+    <>
+      <div className="boxes">
+        <div className="chair-box">
+          <div className="background-color">
+            <img src={img} alt="" />
+          </div>
+        </div>
+        <div className="about">
+          <div className="about-title">
+            <h3>{title}</h3>
+          </div>
+          <div className="color-button">
+            <button className="button-orange"></button>
+            <button className="button-pink"></button>
+            <button className="button-blue"></button>
+          </div>
```

Comment:
you must have an array of available colors (one of yor component's props)
and you must render buttons according to items of this array (and available color must be different for you products of course)